MAUI/DigitalGauge/Display-Types.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: Display Types in Maui Digital Gauge control | Syncfusion
+description: Learn here all about Display Types support in Syncfusion Maui Digital Gauge (SfDigitalGauge) control and more.
+platform: MAUI
+control: DigitalGauge
+documentation: ug
+---
+
+# Display Types in Maui Digital Gauge (SfDigitalGauge)
+
+The digital gauge displays numbers, alphabet, and special characters, which are given in the text property.
+
+## Setting value to number
+
+Numbers can be displayed in digital gauge in different formats using the `CharacterType`property.
```

Comment:
Numbers can be displayed in a digital gauge in different formats using the `CharacterType` property.